modules/test/playwright/tests/api-builder/config.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {devices} from '@playwright/test';
+
+export const config = {
+	name: 'api-builder',
```

Comment:
Also, shouldn't we add this under the `headless-builder-web` folder? :thinking:

Reply:
this only tests API layer, that's why I don't know if the -web is the most suitable